openapi/storecove/client.bal

```diff
@@ -0,0 +1,309 @@
+        http:Request request = new;
+        json jsonBody = check payload.cloneWithType(json);
+        request.setPayload(jsonBody);
+        PreflightInvoiceRecipientResult response = check self.clientEp->post(path, request, targetType=PreflightInvoiceRecipientResult);
+        return response;
+    }
+    # Get InvoiceSubmission Evidence
+    #
+    # + guid - InvoiceSubmission GUID
+    # + return - Success
+    remote isolated function showInvoiceSubmissionEvidence(string guid) returns InvoiceSubmissionEvidence|error {
+        string  path = string `/invoice_submissions/${guid}/evidence`;
+        InvoiceSubmissionEvidence response = check self.clientEp-> get(path, targetType = InvoiceSubmissionEvidence);
+        return response;
+    }
+    # Create a new LegalEntity
+    #
+    # + payload - LegalEntity to create
+    # + return - Success
+    remote isolated function createLegalEntity(LegalEntityCreate payload) returns LegalEntity|error {
+        string  path = string `/legal_entities`;
+        http:Request request = new;
+        json jsonBody = check payload.cloneWithType(json);
+        request.setPayload(jsonBody);
+        LegalEntity response = check self.clientEp->post(path, request, targetType=LegalEntity);
+        return response;
+    }
+    # Get LegalEntity
+    #
+    # + id - legal_entity id
+    # + return - Success
+    remote isolated function getLegalEntity(int id) returns LegalEntity|error {
+        string  path = string `/legal_entities/${id}`;
+        LegalEntity response = check self.clientEp-> get(path, targetType = LegalEntity);
+        return response;
+    }
+    # Delete LegalEntity
+    #
+    # + id - legal_entity id
+    # + return - Success
+    remote isolated function deleteLegalEntity(int id) returns http:Response|error {
+        string  path = string `/legal_entities/${id}`;
+        http:Request request = new;
+        //TODO: Update the request as needed;
+        http:Response response = check self.clientEp-> delete(path, request, targetType = http:Response);
+        return response;
+    }
+    # Update LegalEntity
+    #
+    # + id - legal_entity id
+    # + payload - LegalEntity updates
+    # + return - Success
+    remote isolated function updateLegalEntity(int id, LegalEntityUpdate payload) returns LegalEntity|error {
+        string  path = string `/legal_entities/${id}`;
+        http:Request request = new;
+        json jsonBody = check payload.cloneWithType(json);
+        request.setPayload(jsonBody);
+        LegalEntity response = check self.clientEp->patch(path, request, targetType=LegalEntity);
+        return response;
+    }
+    # Create a new Administration
+    #
+    # + legal_entity_id - The id of the LegalEntity for which to create the Administration
+    # + payload - Administration to create
+    # + return - Success
+    remote isolated function createAdministration(int legal_entity_id, AdministrationCreate payload) returns Administration|error {
```

Comment:
@aneeshafedo Could you please check the reason for generating variable names like `legal_entity_id`. It is supposed to generate in camel cases `legalEntityId`.

Reply:
This is fixed in the latest version.